config/elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    function_naming_convention,
+                    #{regex => "^([a-z]*_?)*(_SUITE)?$"}
```

Comment:
We can update the quotation. Would you mind sending a PR with that change?

When I think of functions with numbers, I'm thinking in stuff like `base64_encode` or `rfc###_…` or `iso6801_…`, etc.